draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -1121,6 +1121,14 @@ Zone owners who do use such a mixed configuration SHOULD mark the RRs with
 without, in order to maximize the likelihood that ECHO will be used in the
 absence of an active adversary.
 
+## Relation to ECHOConfig Extensions
+
+The TLS ECHOConfig structure {{!ECHO}} supports adding arbitrary parameters via
+extensions. This extension mechanism SHOULD NOT be used for any parameter that
+might influence a client's connection properties. In particular, any SVCB parameter
```

Comment:
I think "connection properties" is not clear enough.  To me, the only way an ECHOConfig extension could not affect "connection properties" would be if it were ignored.

Reply:
That may be true. How might you rephrase this?

Reply:
I took a stab at refactoring this. What do you think?